test/AspNet.Security.OAuth.Providers.Tests/Nextcloud/NextcloudTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.Nextcloud
+{
+    public class NextcloudTests : OAuthTests<NextcloudAuthenticationOptions>
+    {
+        public NextcloudTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => NextcloudAuthenticationDefaults.AuthenticationScheme;
+
+        protected override string RedirectUri => "http://localhost/signin-nextcloud";
```

Comment:
How come this one needs the redirect URI to be overridden?

Reply:
ops ... was just for debug reasons. Removed in next commit